### PR TITLE
Task/update dropbox for refresh token

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: react/features/dropbox/functions.web.js [hunk 1]
```diff
@@ -62,7 +52,7 @@ export function _authorizeDropbox(
             return {
                 token: resp.result.access_token,
                 rToken: resp.result.refresh_token,
-                expireDate: getTokenExpiresAtDate(resp.result.expires_in)
+                expireDate: resp.result.expires_in * 1000
```
Comment (horymury) [inline react/features/dropbox/functions.web.js:55]: this is the number of seconds until token expires , <=> 4 hrs, not the timestamp in seconds

Comment (horymury) [reply react/features/dropbox/functions.web.js:55]: new Date(Date.now() + ( resp.result.expires_in* 1000)).getTime()

**Review by horymury — CHANGES_REQUESTED** [review]

File: react/features/dropbox/functions.native.js [hunk 2]
```diff
@@ -7,16 +7,20 @@ const { Dropbox } = NativeModules;
 /**
  * Action to authorize the Jitsi Recording app in dropbox.
  *
- * @param {string} appKey - The Jitsi Recorder dropbox app key.
- * @param {string} redirectURI - The return URL.
  * @returns {Promise<string>} - The promise will be resolved with the dropbox
  * access token or rejected with an error.
  */
 export function _authorizeDropbox(): Promise<string> {
```
Comment (horymury) [inline react/features/dropbox/functions.native.js:13]: promise<Object> ?

**Comment by horymury** [comment]
@titusmoldovan   please check all JSdocs where I describe expireDate and change the description, since now it s time stamp and not ISO string. ty

File: android/sdk/src/main/java/org/jitsi/meet/sdk/DropboxModule.java [hunk 3]
```diff
@@ -181,10 +183,15 @@ public void onHostPause() {}
...
     @Override
     public void onHostResume() {
-        String token = Auth.getOAuth2Token();
-
-        if (token != null && this.promise != null) {
-            this.promise.resolve(token);
+        DbxCredential credential = Auth.getDbxCredential();
+
+        if (credential != null && this.promise != null) {
+            WritableMap result = new WritableNativeMap();
```
Comment (saghul) [inline android/sdk/src/main/java/org/jitsi/meet/sdk/DropboxModule.java:189]: Use Arguments.createMap() instead. Seems to be the convention.

**Review by saghul — COMMENTED** [review]
Small nit, LGTM otherwise.

File: android/sdk/src/main/java/org/jitsi/meet/sdk/DropboxModule.java [hunk 4]
```diff
@@ -20,6 +21,7 @@
 import com.dropbox.core.android.Auth;
 import com.facebook.react.bridge.ReactMethod;
 import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.bridge.WritableNativeMap;
```
Comment (saghul) [inline android/sdk/src/main/java/org/jitsi/meet/sdk/DropboxModule.java:24]: You can remove this import now.

**Review by horymury — APPROVED** [review]

**Review by horymury — APPROVED** [review]

**Pull request merged**